OpenRA.Renderer.Sdl2/VertexBuffer.cs

```diff
@@ -48,6 +48,16 @@ public void SetData(T[] data, int start, int length)
 			ErrorHandler.CheckGlError();
 		}
 
+		public void SetData(IntPtr data, int start, int length)
+		{
+			Bind();
```

Comment:
@pchote does your GL implementation expose ARB_vertex_array_object in a legacy context, or do we need to go core to have VAOs? I'd rather not use APPLE_vertex_array_object, its semantics are subtly bogus.